ruby/test/fixtures/log/flaky_tests.txt

```diff
@@ -0,0 +1 @@
+["BTest#test_bar","ATest#test_flaky_fails_retry","ATest#test_flaky","ATest#test_bar"]
```

Comment:
Is this used?

Reply:
Oops no, that is the file generated after test execution.